01-js-intro-and-jquery/js/javascript.js

```diff
@@ -0,0 +1,84 @@
+
+
+var functionFadeToggleSection = (function() {
+  //Private variables
+  var selectedSection = $('section.hidden-section');
+  var text = $('input.alias');
+
+  var callBackFunction = function() {
+    text.trigger('focus'); //only in chrome?
+    
+  };
+
+  var ft = function() {
+    selectedSection.fadeIn("slow",callBackFunction);
+  };
+  
+  //public API
+  return {
+    fadeAndToogle: ft
+
+  };//end return 
+  
+})();
+
+
+var removeClassFunction = (function() {
```

Comment:
From my perspective, this function is unnecessary, since you could just call jQuery's ``removeClass`` directly where you need (you don't really need to encapsulate any data being used here).